File: src/main/java/com/fish/entity/pojo/Accepted.java
package com.fish.entity.pojo;

import com.mybatisflex.annotation.Id;
import com.mybatisflex.annotation.Table;
import jakarta.validation.constraints.NotNull;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;

/**
 *  实体类。
 *
 * @author fish
 * @since 2023-11-14
 */
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Table(value = "oj_accepted")
public class Accepted implements Serializable {
    @Id
    @NotNull(message = "用户Id未填写")
    private String userId;
    @Id
    @NotNull(message = "题目Id未填写")
    private Long topicId;
}

File: src/main/kotlin/com/fish/entity/pojo/Accepted.kt
package com.fish.entity.pojo

import com.mybatisflex.annotation.Id
import com.mybatisflex.annotation.Table
import jakarta.validation.constraints.NotNull
import java.io.Serializable

/**
 * 实体类。
 *
 * @author fish
 * @since 2023-11-14
 */
@Table(value = "oj_accepted")
open class Accepted: Serializable {
    @Id var userId: @NotNull(message = "用户Id未填写") String? = null
    @Id var topicId: @NotNull(message = "题目Id未填写") Long? = null
}
